src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1483,6 +1488,14 @@ internal static void ApplyConfiguration(JObject config, ScriptHostConfiguration
                 }
             }
 
+            if (config.TryGetValue("lazyLoadExtensions", out value))
```

Comment:
Recommend naming this "loadReferencedExtensionsOnly" since that's closer in meaning to what we're actually doing. We're not really doing a lazy load.